src/arch/dotproductavx.cpp

```diff
@@ -16,8 +16,8 @@
 ///////////////////////////////////////////////////////////////////////
 
 #if !defined(__AVX__)
-#error Implementation only for AVX capable architectures
-#endif
+  //#error Implementation only for AVX capable architectures
```

Comment:
```suggestion
#if defined(__x86_64__)
#error Implementation only for AVX capable architectures
#endif
```

Reply:
Yup. Working on that now. As I understand it __x86_64__ is only defined when building for 64bit systems. SSE, AVX2 etc work just fine on 32bit systems too - should we be using:

#if defined(__x86__64) || defined(something_else)

to capture both cases?

Reply:
That's correct, but i386 systems are getting rarer, and the normal Tesseract build handles them correctly.  Nevertheless I updated my suggestion above to handle `__i686__`, too.

Reply:
Updated. Please let me know if you'd like any further changes. Thanks.